src/qilib/configuration_helper/adapters/common_instrument_adapter.py

```diff
@@ -63,3 +61,16 @@ def _filter_parameters(self, parameters: PythonJsonStructure) -> PythonJsonStruc
             PythonJsonStructure: Contains the instrument snapshot without the instrument
                                  parameters which are filtered out by this function.
         """
+
+    def __raise_none_value_parameters(self, config: PythonJsonStructure, parameters: List[dict]) -> None:
+        """ Raises a ValueError if non value parameters are present in the configuration.
```

Comment:
Why the term 'non value parameter'? It should either be `None value parameter` (meaning a value `None`), or `non-value parameter`, but then it should be explained what it means (e.g. None value, or (None or np.NaN) or something else.

In the code below also `non_value_parameters` should be updated to `none_value_parameters`

Reply:
Yes, indeed 😊.